documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -30,7 +30,7 @@
 
 ### Resolved issues
 
-* 
+* (IDETECT-4746) – Fixed incorrect labeling of transitive dependencies as direct by the Cargo Lockfile Detector.
```

Comment:
My apologies for the delay, when I read the line back over I thought of this rewording idea:

* (IDETECT-4746) – Fixed Cargo Lockfile Detector incorrectly labeling transitive dependencies as direct dependencies.